### DISCUSSION TITLE
Fully hidden collection

### DISCUSSION THREAD

**Discussion by sksar** [pocketbase/pocketbase#6161] [Q&A] [answered]
Is it possible to hide a collection created using golang code / javascript code. 
I am extending pocketbase using golang, and creating a internal table / collection. 
But I don't want this table / collection to be accessible from anywhere, neither the Admin UI, nor the API or whatever. 
It will only be accessible from the golang code.

Is this possible, and if so how ?

**Comment by ganigeorgiev** [comment, marked as answer]
If it is a collection - No, but you can create a plain SQLite table and manage it manually with the [dbx abstraction](https://github.com/pocketbase/dbx) if that's really what you want.

What is the problem you are trying to solve?

**Reply by ganigeorgiev** [reply]
Other than what I already posted, I have no other suggestions.

I still don't exactly understand what is your use case, nor what you are trying to do in general, but note that there are no plans to change or complicate further how the current tokens are generated or verified.
